### PR TITLE
feat: tag analytical cluster

### PR DESCRIPTION
**Pull request by wai-wong-edb**

### CODE REVIEW

File: examples/resources/biganimal_analytics_cluster/aws/resource.tf [hunk 1]
```diff
@@ -32,6 +32,16 @@ resource "biganimal_analytics_cluster" "analytics_cluster" {
   project_id   = var.project_id
   pause        = false
 
+  #tags = [
+  #  {
+  #     tag_name  = "wai-tag-3"
```
Comment (vishalsawale9) [inline examples/resources/biganimal_analytics_cluster/aws/resource.tf:37]: Better give some other name :)

File: pkg/provider/data_source_analytics_cluster.go [hunk 2]
```diff
@@ -193,6 +193,23 @@ func (r *analyticsClusterDataSource) Schema(ctx context.Context, req datasource.
 					"Pausing a high availability cluster shuts down all cluster nodes",
 				Optional: true,
 			},
+			"tags": schema.SetNestedAttribute{
+				Description: "Assign existing tags or create tags to assign to this resource",
```
Comment (vishalsawale9) [inline pkg/provider/data_source_analytics_cluster.go:197]: https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/582/files#r1728390020

**Review by vishalsawale9 — COMMENTED** [review]
A few neat comments, but it looks good to me overall.

File: pkg/provider/data_source_analytics_cluster.go (hunk 2 above)
Comment (edbamolvyavahare) [inline pkg/provider/data_source_analytics_cluster.go:197]: as this is a data_source description, we should not mention assign or create, instead it should mention something like "show tags associated with this resource"

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Pull request merged**